CLI/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+setup(name='CLI',
+      version='0.0.0',
+      py_modules= ['cli'],
+      install_requires=[
+            'Click',
```

Comment:
This didn't seem to install the package. Adding a line to `Pipfile` silenced the error.

Reply:
Added to pipfile the way Samir suggested.